modules/nixos/monitoring/loki.nix

```diff
@@ -0,0 +1,15 @@
+{
+  services.telegraf.extraConfig.inputs.prometheus.urls = [
+    "http://localhost:3100/metrics"
+  ];
+
+  services.loki = {
```

Comment:
It took me a while to find sane loki configuration:

https://github.com/Mic92/dotfiles/blob/c912c73266f10973106a8522cbec827e2802918d/nixos/eva/modules/loki.nix#L39